src/containers/TaxonomyVersions/TaxonomyVersionsPage.tsx

```diff
@@ -60,14 +60,14 @@ const TaxonomyVersionsPage = () => {
       </Row>
       {!!showNewForm && (
         <NewFormWrapper>
-          <VersionForm existingVersions={data ?? []} onClose={() => setShowNewForm(false)} />
+          <VersionForm existingVersions={data ?? []} onClose={() => setShowNewForm(false)} titleElement="h2" />
```

Comment:
Tror vi har kalt dette `headingLevel` andre steder. Stemmer for at vi fortsetter med det!